docs/get-started/installation.md

```diff
@@ -150,6 +150,46 @@ docker run --rm quickwit/quickwit --version
 # You can also safely ignore jemalloc warnings.
 docker run --rm --platform linux/amd64 quickwit/quickwit --version
 ```
+Here is the directory layout of Quickwit inside the container in case you need to mount a volume:
+
+```bash
+/quickwit
+    ├── config
+    │   └── quickwit.yaml
+    ├── quickwit
+    └── qwdata
+```
+
+- `config/quickwit.yaml`: the default configuration file.
+- `quickwit`: the quickwit executable binary.
+- `qwdata/`: the default data directory.
+
+Here is a list of the available environment variables and ports options that can be used to configure Quickwit:
+
+[Node configuration](../configuration/node-config.md):
```

Comment:
It feels weird to put this list without defining the variables. I would definitely avoid that and put just the link which is a very good idea.